io.embrace.sdk/Editor/Editor Windows/Installation.cs

```diff
@@ -184,36 +188,29 @@ public static void SetupManifest(EmbraceSdkInfo embraceSdkInfo)
                     }
                 }
 
-                if (!hasEmbraceScopedRegistry)
+                if (hasEmbraceScopedRegistry)
                 {
-                    List<string> scopes = new List<string>() { "io.embrace" };
-                    ScopedRegistry embraceRegistry = new ScopedRegistry(package.name, embraceSdkInfo.npmAPIEndpoint, scopes.ToArray());
+                    // We have the scoped registry.
+                    // We now need to remove it
+                    var embraceScopedRegistryEntry = scopedRegistries.FirstOrDefault(content => (string)content["name"] == "io.embrace");
+                    if (embraceScopedRegistryEntry != null)
+                    {
+                        scopedRegistries.Remove(embraceScopedRegistryEntry);
```

Comment:
seems like we could maybe move this to after line 186 above, then the check + `FirstOrDefault` wouldn't be needed since we have the right `content` at that point already

Reply:
Makes sense to me!